### DISCUSSION TITLE
Google Brain 推出新的文字轉圖像模型──Imagen

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#640] [General]
https://technews.tw/2022/05/28/google-brain-imagen/

Google 推出一個AI模組--Imagen。透過輸入想要輸出圖像的文字敘述，例如：一隻戴著藍色格子貝雷帽和穿著紅色圓點高領毛衣的黃金獵犬。Imagen就會利用文本條件超解析度擴散模型輸出最高像素1024*1024符合文字敘述的圖像。

**Comment by 11timyyy** [comment]
可以真實體現出腦中所想的畫面，感覺很新奇

**Comment by Rosancl** [comment]
前幾天有在限時動態看到朋友在玩這個，就生成的畫面來說真的比前幾年看過的類似程式逼真很多~~ 👍

**Comment by stan890425** [comment]
用程式畫畫ㄟ~但是怕人類教壞他
https://www.inside.com.tw/article/27854-imagen-and-ai-ethic

**Comment by Ben6uo** [comment]
如果可以用圖片來生成6位數就更加好了

**Comment by liangyinglly** [comment]
這真得的圖像辨識的強大運用，超級喜歡這個點子，想深入了解他的運作方式

**Comment by gnsJhenJie** [comment]
很酷的科技! 不知道這樣的技術能應用在什麼場景

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by jason79461385** [comment]
感覺可以畫出一些很meme的東西 蠻有趣的
